vtStorScsi/ScsiCommandExtensions.cpp

```diff
@@ -148,18 +149,23 @@ namespace vtStor
         eErrorCode cScsiCommandExtensions::IssueCommand_AtaReadDma(std::shared_ptr<IDrive> Drive, U32 CommandType, std::shared_ptr<IBuffer> Data, U32 Lba, U16 Count)
```

Comment:
With the AtaPassThrough, the parameter is more readable instead of filling a lot of weird values. It's easier for maintenance